crates/dbsp/src/operator/time_series/rolling_aggregate.rs

```diff
@@ -120,36 +117,7 @@ where
             )
             .typed()
     }
-}
-
-impl<PK, TS, V> Stream<RootCircuit, OrdIndexedZSet<PK, Tup2<TS, V>>>
-where
-    PK: DBData,
-    TS: DBData + PrimInt,
-    V: DBData,
-{
-    pub fn as_partitioned_zset(
-        &self,
-    ) -> Stream<RootCircuit, OrdPartitionedIndexedZSet<PK, TS, DynDataTyped<TS>, V, DynData>> {
-        let factories = BatchReaderFactories::new::<PK, Tup2<TS, V>, ZWeight>();
-
-        self.inner()
-            .dyn_map_index(
-                &factories,
-                Box::new(|(k, v), kv| {
-                    kv.from_refs(k, unsafe { v.downcast::<Tup2<TS, V>>().erase() })
-                }),
-            )
-            .typed()
-    }
-}
 
-impl<PK, TS, V> Stream<RootCircuit, OrdPartitionedIndexedZSet<PK, TS, DynDataTyped<TS>, V, DynData>>
-where
-    PK: DBData,
-    TS: DBData + UnsignedPrimInt,
```

Comment:
I liked this constraint, is there some place to put it?

Reply:
good point, let me bring it back